rules-reviewed/eap7/eap71/resteasy30-36.windup.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="resteasy30-36"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of applications that use RESTEasy 3.0 and may require
+            individual attention when migrating to RESTEasy 3.6.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[7,8)" />
+        <targetTechnology id="resteasy" versionRange="[3,)" />
+    </metadata>
+    <rules>
+        <!--https://issues.jboss.org/browse/WINDUPRULE-370 -->
+        <rule id="resteasy30-36-00001">
+            <when>
+                <project>
+                    <artifact groupId="org.jboss.spec.javax.ws.rs" artifactId="jboss-jaxrs-api_2.0_spec" />
+                </project>
+            </when>
+            <perform>
+                <hint title="RESTEasy 3.6 - Resource Method Algorithm Switch changes" effort="1" category-id="potential">
+                    <message>If you migrate your application from JBoss EAP from 7.1.1 to 7.2.x and want to retain the stricter behavior as defined in the JAX-RS 2.0 specification, set the `jaxrs.2.0.request.matching` option to 'true'.</message>
```

Comment:
Sorry to still bother you with this but please apply this ` char to have the right formatting.
```suggestion
                    <message>If you migrate your application from JBoss EAP from 7.1.1 to 7.2.x and want to retain the stricter behavior as defined in the JAX-RS 2.0 specification, set the `jaxrs.2.0.request.matching` option to `true`.</message>
```